packages/cli-lib/projectsWatch.js

```diff
@@ -120,18 +120,11 @@ const createNewBuild = async (accountId, projectName) => {
     const { buildId } = await provisionBuild(accountId, projectName);
     return buildId;
   } catch (err) {
-    if (
-      err.error.subCategory === 'PipelineErrors.PROJECT_MANAGED_THROUGH_GITHUB'
-    ) {
-      logApiErrorInstance(err, new ApiErrorContext({ accountId, projectName }));
-      process.exit(1);
-    } else if (err.error.subCategory === 'PipelineErrors.PROJECT_LOCKED') {
-      logger.error(i18n(`${i18nKey}.errors.projectLocked`));
-    } else {
-      logApiErrorInstance(err, new ApiErrorContext({ accountId, projectName }));
+    logApiErrorInstance(err, new ApiErrorContext({ accountId, projectName }));
```

Comment:
I tried to get rid of any reference to the subcategory types, but we still need to check for the locked category type unfortunately. We need to make sure that we don't cancel the current staged build if a project is locked b/c a locked project means that some other process is currently running the watch command and we don't want to interrupt it.